File: CyberLife_Calendar/src/db/pojo/eventPOJO/EventEndSchedule.java
package db.pojo.eventPOJO;

import java.sql.Date;

public class EventEndSchedule {
	
	private int cod_fim_repeticao;
	private Date dia_fim;
	private int qtd_recorrencias;
	private int fk_evento;

	public int getCod_fim_repeticao() {
		return cod_fim_repeticao;
	}

	public void setCod_fim_repeticao(int cod_fim_repeticao) {
		this.cod_fim_repeticao = cod_fim_repeticao;
	}

	public Date getDia_fim() {
		return dia_fim;
	}

	public void setDia_fim(Date dia_fim) {
		this.dia_fim = dia_fim;
	}

	public int getQtd_recorrencias() {
		return qtd_recorrencias;
	}

	public void setQtd_recorrencias(int qtd_recorrencias) {
		this.qtd_recorrencias = qtd_recorrencias;
	}

	public int getFk_evento() {
		return fk_evento;
	}

	public void setFk_evento(int fk_evento) {
		this.fk_evento = fk_evento;
	}

}

File: API/JSON_objects/event/EventEndSchedule.php
<?php
    class EventEndSchedule implements \JsonSerializable {

        private $cod_fim_repeticao;
        private $dia_fim;
        private $qtd_recorrencias;
        private $fk_evento;
    
        public function get_cod_fim_repeticao() : int {
            return $this->cod_fim_repeticao;
        }

        public function set_cod_fim_repeticao(int $cod_fim_repeticao){
            $this->cod_fim_repeticao = $cod_fim_repeticao;
        }

        public function get_dia_fim() : int {
            return $this->dia_fim;
        }

        public function set_dia_fim(int $dia_fim){
            $this->dia_fim = $dia_fim;
        }

        public function get_qtd_recorrencias() : int {
            return $this->qtd_recorrencias;
        }

        public function set_qtd_recorrencias(int $qtd_recorrencias){
            $this->qtd_recorrencias = $qtd_recorrencias;
        }

        public function get_fk_evento() : int {
            return $this->fk_evento;
        }

        public function set_fk_evento(int $fk_evento){
            $this->fk_evento = $fk_evento;
        }

        public function jsonSerialize()
        {
            $vars = get_object_vars($this);
            return $vars;
        }
    }
?>